custodian/qchem/jobs.py

```diff
@@ -240,36 +293,47 @@ def opt_with_frequency_flattener(
-                    print("All frequencies positive!")
-                    break
-                if abs(outdata.get("frequencies")[0]) < 15.0 and outdata.get("frequencies")[1] > 0.0:
-                    print("One negative frequency smaller than 15.0 - not worth further flattening!")
-                    break
-                if len(energy_history) > 1:
-                    if abs(energy_history[-1] - energy_history[-2]) < energy_diff_cutoff:
-                        print("Energy change below cutoff!")
+                if not transition_state:
+                    if outdata.get("frequencies")[0] > 0.0:
+                        print("All frequencies positive!")
                         break
-                opt_QCInput = QCInput(
-                    molecule=opt_outdata.get("molecule_from_optimized_geometry"),
-                    rem=opt_rem,
-                    opt=orig_input.opt,
-                    pcm=orig_input.pcm,
-                    solvent=orig_input.solvent,
-                    smx=orig_input.smx,
-                )
-                opt_QCInput.write_file(input_file)
+                    elif abs(outdata.get("frequencies")[0]) < 15.0 and outdata.get("frequencies")[1] > 0.0:
+                        print("One negative frequency smaller than 15.0 - not worth further flattening!")
+                        break
+                    if len(energy_history) > 1:
+                        if abs(energy_history[-1] - energy_history[-2]) < energy_diff_cutoff:
+                            print("Energy change below cutoff!")
+                            break
+                    opt_QCInput = QCInput(
+                        molecule=opt_outdata.get("molecule_from_optimized_geometry"),
+                        rem=opt_rem,
+                        opt=orig_input.opt,
+                        pcm=orig_input.pcm,
+                        solvent=orig_input.solvent,
+                        smx=orig_input.smx,
+                    )
+                    opt_QCInput.write_file(input_file)
+                else:
+                    if outdata.get('frequencies')[0] < 0.0 and outdata.get('frequencies')[1] > 0.0:
+                        print("Saddle point found!")
+                        break
+                    elif abs(outdata.get("frequencies")[1]) < 15.0 and outdata.get("frequencies")[2] > 0.0:
+                        print("Second small imaginary frequency (smaller than 15.0) - not worth further flattening!")
+                        break
+                    else:
+                        opt_QCInput = QCInput(
+                            molecule=opt_outdata.get("molecule_from_optimized_geometry"),
+                            rem=opt_rem,
+                            opt=orig_input.opt,
+                            pcm=orig_input.pcm,
+                            solvent=orig_input.solvent,
+                            smx=orig_input.smx)
+                        opt_QCInput.write_file(input_file)
```

Comment:
Is there a particular reason you are using `print` statements here rather than, e.g. `warnings.warn` / `warnings.info` ? It seems like it might be useful to emit warnings instead

Reply:
I tried to adhere to the structure originally written by @samblau. Before I embarked on this PR, I spoke with him about this function. We agreed that it eventually needed a full overhaul, but for now, I'd just add functionality. I agree that eventually, providing a formal warning is a good move.

Reply:
I would support switching print statements to warnings.

Reply:
Added